recipes/icu/bld.bat

```diff
@@ -0,0 +1,15 @@
+if "%ARCH%"=="32" (
```

Comment:
Maybe.  msbuild might take care of that, but I'm not sure.  Need to build this myself and see.  When I wrote this, I certainly was aware of those approaches, so it might be right here.